src/vm/exceptionhandling.cpp

```diff
@@ -4635,33 +4635,43 @@ VOID DECLSPEC_NORETURN UnwindManagedExceptionPass1(PAL_SEHException& ex, CONTEXT
     EEPOLICY_HANDLE_FATAL_ERROR(COR_E_EXECUTIONENGINE);
 }
 
-VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex)
+VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex, bool isHardwareException)
```

Comment:
Yeah. That looks better to me as well.

I'll change the alignment as well. (my vim has been configured for Linux kernel. :( I might need to get it dual-configured)